modules/Blockchain/Readme.md

```diff
@@ -0,0 +1,119 @@
+Blockchain implementations
+========================
+
+#### Instruction manual for enabling new blockchain integrations
+
+OT-Node operates as a multi-chain node in the network and can communicate with different 
+blockchain simultaneously. Currently, it supports Ethereum, xDai, Polygon, while OriginTrail Parachain is in experimental phase.
+In order to integrate new blockchain implementation, the following modifications have to be performed.
+
+## Directory structure
+
+Create directory for new blockchain using the following command:
+```bash
+mkdir modules/Blockchain/<blockchain_name>
+```
+If the existing web3 implementation is compatible with the new blockchain, copy content of Ethereum directory to the new directory.
```

Comment:
maybe to add else xD What if the web3 implementation is not compatible with new blockchain? Then contact us for more info or something else...